src/main/c/sslcontext.c

```diff
@@ -1591,6 +1591,27 @@ TCN_IMPLEMENT_CALL(jboolean, SSLContext, setSessionIdContext)(TCN_STDARGS, jlong
     }
     return JNI_FALSE;
 }
+
+TCN_IMPLEMENT_CALL(jint, SSLContext, setMode)(TCN_STDARGS, jlong ctx, jint mode)
+{
+    tcn_ssl_ctxt_t *c = J2P(ctx, tcn_ssl_ctxt_t *);
+
+    UNREFERENCED(o);
```

Comment:
Can you explain what this does? In some cases we do this after the null check/return.

https://github.com/netty/netty-tcnative/blob/1.1.33/src/main/c/ssl.c#L1600